src/Domain/Tags.hpp

```diff
@@ -312,6 +316,26 @@ struct InterfaceMesh : db::ComputeTag, Tags::Mesh<VolumeDim - 1> {
   using volume_tags = tmpl::list<Mesh<VolumeDim>>;
 };
 
+/// \ingroup DataBoxTagsGroup
+/// \ingroup ComputationDomainGroup
+/// Computes the coordinates in the frame `Frame` on the faces defined by
+/// `Direction`. Intended to be prefixed by a `Tags::InterfaceComputeItem` to
+/// define the directions on which to compute the coordinates.
+template <size_t VolumeDim, typename Frame = ::Frame::Inertial>
+struct BoundaryCoordinates
```

Comment:
It would be nice if this could be done using the standard Coordinates tag, but I'm not sure if that works because of the mismatch between the manifold and tensor dimensions.

Reply:
I don't know what `Face` means, assuming it's interface, I'd prefer that to be consistent with the way `Interface` is used in tags below.

Reply:
#1012